### DISCUSSION TITLE
Max pinned tabs

### DISCUSSION THREAD

**Discussion by BuruY** [zen-browser/desktop#3816] [General]
I really like the ideas of workspaces, shared tabs and pinned tabs.

I am now running into a weird issue where these seems to be a limit of pinned tabs, and I am unable to change that.
It really is confisuing to have pinned tabs 'hidden' like this
![image](https://github.com/user-attachments/assets/f348c380-84be-473a-96e6-38110f0b4cc6)

Not sure if the image is clear, but basically the 📦 is my workspace and the next icon visible is Grafana, but that is not my top pinned tab (see scroll bar), it is very confusing and I would like to be able to resize the pinned tab bar section.

**Comment by thothi** [comment]
Yeah for some reason when pinning more than 6 tabs, you are forced to now scroll in the pinned tab bar. Really annoying as I pin my tabs to always see them, without need for scrolling...

**Comment by vraravam** [comment]
you can use this css in your `userChrome.css` file to expand the pinned tab bar:
```
/* Elongate the pinned tabs section */
#vertical-pinned-tabs-container {
  /* max-height: calc(15 * var(--tabstrip-min-height) + var(--space-large)) !important; */
  max-height: none !important;
}
```

**Reply by BuruY** [reply]
Might be a weird question, but where is this file located? I cannot find it in de default profile folder.

**Reply by vraravam** [reply]
inside of the profile folder, you will need to create a folder called `chrome`, and within that the filename is `userChrome.css` (please be mindful of the caps - that's important i would think.

**Reply by BuruY** [reply]
![image](https://github.com/user-attachments/assets/d994529b-52da-4216-8c00-600941b0af85)

Does not seem to work.

Now when starting Zen it always defaults to safe mode

**Reply by vraravam** [reply]
<img width="291" alt="Screenshot 2024-12-19 at 19 03 05" src="https://github.com/user-attachments/assets/450bcb06-3d36-4155-a598-fbcccf47a649" />
As you can see - I have that same CSS and I am able to increase the number of tabs in the pinned section. I am running the latest twilight version. Please re-verify the file location and name.

**Reply by BuruY** [reply]
You can see the location + filename in the screenshot I posted, are they incorrect?

**Reply by vraravam** [reply]
are you sure that this is the profile that's being used? the `xb8....(alpha)`?
in my system, this is the location and the content of the file:
<img width="1031" alt="Screenshot 2024-12-19 at 21 25 07" src="https://github.com/user-attachments/assets/761709da-15ab-40de-965b-a7e002c55acc" />

**Reply by vraravam** [reply]
and this is the `profiles.ini` that I use:
<img width="840" alt="Screenshot 2024-12-19 at 21 27 04" src="https://github.com/user-attachments/assets/b4725af9-28f0-4ccf-a693-7dbe2d290f0b" />
So, the relative path of the `profiles.ini` vs the `Profiles` folder needs to be correct.
